packages/lesswrong/lib/collections/comments/views.js

```diff
@@ -7,14 +7,21 @@ import { ensureIndex,  combineIndexWithDefaultViewIndex} from '../../collectionU
 // Auto-generated indexes from production
 
 Comments.addDefaultView(terms => {
-  const validFields = _.pick(terms, 'userId');
+  const validFields = _.pick(terms, 'userId', 'authorIsUnreviewed');
   const alignmentForum = getSetting('forumType') === 'AlignmentForum' ? {af: true} : {}
+  let hideUnreviewedAuthorComments = {}
+  if (getSetting('hideUnreviewedAuthorComments')) {
+    // allow for null so that comments that haven't had the default value set
+    // yet (ie: those created by the frontend immediately prior appear
+    hideUnreviewedAuthorComments = {authorIsUnreviewed: {$ne: true}}
```

Comment:
This should use the `viewFieldAllowAny` variable that you can import from vulcan-lib.

Reply:
At first I thought you meant that the {$ne: true} should be replaced by that, which wouldn't work, because we specifically want to exclude those with authorIsUnreviewed = true. Then I thought you meant that below where we override this setting we should use viewFieldAllowAny, which would make sense. Then I found the problem that caused this todo in posts/views

```
// TODO: fix vulcan so it doesn't do deep merges on viewFieldAllowAny
```

😑 

Realizing that I was doing a non-idiomatic version of a trinary did lead me to fix that, test the result again, then realize that validFields was not overriding the defaults.